### ISSUE TITLE
Add Service Grid

### ISSUE THREAD

**Issue by nilmerg** [Icinga/icingadb-web#118]
![Screenshot from 2020-08-10 13-33-30](https://user-images.githubusercontent.com/16668527/89778619-21202000-db0e-11ea-8798-ace20be4356c.png)

**Comment by flourish86** [comment]
Here are the refined Mockups for the redesigned Service Grid View.

![Desktop Copy 5](https://user-images.githubusercontent.com/6977434/97329102-fcb72280-1876-11eb-9f0f-54020678d32c.jpg)

The »Joystick Navigation« was replaced by a bird's eye navigation, which offers map like viewport customization.

![Desktop Copy 4](https://user-images.githubusercontent.com/6977434/97329204-1b1d1e00-1877-11eb-926e-e1f08fec5d64.jpg)

**Comment by flourish86** [comment]
Interacting with the bird's eye navigation.

![Artboard Copy](https://user-images.githubusercontent.com/6977434/97329323-37b95600-1877-11eb-9ca6-bb990fbde609.jpg)


![Artboard](https://user-images.githubusercontent.com/6977434/97329372-4142be00-1877-11eb-9daf-2b5adc921cf5.jpg)


![Artboard Copy 2](https://user-images.githubusercontent.com/6977434/97329472-5c153280-1877-11eb-952a-a83e8ad88e9e.jpg)

**Comment by flourish86** [comment]
Here's a little code example on how to do the rounded corner buttons in css.

### HTML

```
<table>  
  <thead>
    <tr>
      <td style="text-align: right;"><a class="corner-btn nw">↖️</a></td>
      <td><a class="nav-btn">Navigate top</a></td>
      <td><a class="corner-btn ne">↗️</a></td>
    </tr>
  </thead>
</table>
```

### CSS (Less)

```
@body-bg: lavender;
@btn-thickness: 36px;
@btn-inner-corner-radius: 24px;

* {
  box-sizing: border-box;
}

body {
  background: @body-bg;
}

table {
  width: 100%;
}

table td {
  padding: 0;
  vertical-align: top;
}

a {
  text-decoration: none;
}

.nav-btn {
  display: block;
  background: maroon;
  color: white;
  text-align: center;
  height: @btn-thickness;
}

a.corner-btn {
  display: inline-flex;
  justify-content: center;
  align-items: center;
  
  width: @btn-thickness + @btn-inner-corner-radius;
  height: @btn-thickness + @btn-inner-corner-radius;
  background: maroon;
  position: relative;
  
  &:before {
    content: "";
    display: block;
    width: @btn-inner-corner-radius;
    height: @btn-inner-corner-radius;
    background: @body-bg;
    position: absolute;
  }
  
   
  &.nw {
    border-top-left-radius: @btn-thickness + @btn-inner-corner-radius;
    
    &:before {
      border-top-left-radius: @btn-inner-corner-radius;
      bottom: 0;
      right: 0;
    }
  }
  
  &.ne {
    border-top-right-radius: @btn-thickness + @btn-inner-corner-radius;
    
    &:before {
      border-top-right-radius: @btn-inner-corner-radius;
      bottom: 0;
      left: 0;
    }
  }
}
```

**Comment by nilmerg** [comment]
@flourish86 This will be closed by #204. Please create a new issue for the re-design and add your mockups/examples there. Thank you!
